### PR TITLE
Added Project Plan

### PR DESCRIPTION
**Pull request by DaitTan**
#3

### CODE REVIEW

File: Documentation/Project Plan/Project Plan.tex [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+01/10 & 08/10 & Prototype Design & \begin{tabular}[c]{@{}l@{}}A simple non-coded functioning app for \\ reviewing by Customers\end{tabular} \\ \hline
+09/10 & 15/10 & Prototype Design Review by Customers & \begin{tabular}[c]{@{}l@{}}A review on the changes that can be made to \\ design to improve user satisfaction\end{tabular} \\ \hline
+16/10 & 10/11 & Backend and Frontend Coding and Unit Testing & \begin{tabular}[c]{@{}l@{}}App for scrap dealers and a webapp of \\ an e-commerce store\end{tabular} \\ \hline
+11/11 & 15/11 & Testing & A app that has minimum bugs. \\ \hline
+\end{tabular}
+\end{table}
+\section{Team Structure}
+The team structure followed is \textbf{Chief Programmer Team}.
+We have one chief programmer to whom the team members report. Apart from that, we have a librarian who maintains all the documentation and communication related work.
+\par The Chief Programmer is Aakash Aggarwal. Megana Ganpathiraju and Madhukar Jaisswal, Vikesh Meena and Tanmay Khandait fall under him.
```
Comment (AnilSharma1997) [inline Documentation/Project Plan/Project Plan.tex]: Add my name"Anil Sharma"

File: Documentation/Project Plan/Project Plan.tex [hunk 2]
```diff
@@ -84,17 +84,28 @@ \section{Deliverables Deadlines}
...
+The team structure followed is a blend of \textbf{Chief Programmer Team} and \textbf{Egoless team.}
+\par Structurally, we follow the chief programmer team structure with a chief programmer leading, reviewing and deciding the flow of work, and a librarian whose main job is to maintain the documentation.
+\par However, every member has the freewill to work in any part of the project in addition to his assigned role. Everyone contributes and provide feedback, both positive and negative, to other team members. The views of every member are taken into consideration before taking any step related to the project.
+
+\par Mainly, for the design phase of the project, we have divided the team into two parts. First group, which handles the front end prototype design, consists of Akash Agrawal and Megana Ganpathiraju. And the second, those who handle the back end database design, has Madhukar Jaiswal and Anil Sharma. Three, those who make sure all the related documentation is ready, comprises of Tanmay Khandait and Vikesh Meena.
```
Comment (akash1997) [inline Documentation/Project Plan/Project Plan.tex]: Change the three part to third

File: Documentation/Project Plan/Project Plan.tex [hunk 3]
```diff
@@ -84,17 +84,28 @@ \section{Deliverables Deadlines}
 11/11 & 15/11 & Testing & A app that has minimum bugs. \\ \hline
 \end{tabular}
 \end{table}
+
+\section{Software Engineering Model}
+The software model which we plan on using is Prototype Model. The main aim of the model is to gather the requirements and then design, build a prototype, evaluate and redesign the product as per the stakeholder's requirements. 
+Considering the scope and scalability of our idea, it needs to go through a number of iterations of the prototype model in order to come up with a stable system, both technically and economically.
+\begin{center}
+    \includegraphics[scale=1]{Images/Prototype.jpg}
```
Comment (akash1997) [inline Documentation/Project Plan/Project Plan.tex]: Add source of image below the image

File: Documentation/Project Plan/Project Plan.tex (hunk 2 above)
Comment (akash1997) [inline Documentation/Project Plan/Project Plan.tex]: Add the coding part division too along with the design division

**Pull request merged**